server/models/blogs.js

```diff
@@ -0,0 +1,34 @@
+import mongoose from 'mongoose';
+
+const { Schema } = mongoose;
+
+const BlogsSchema = new Schema({
+  userId: {
+    type: Schema.Types.ObjectId,
+    ref: 'User',
+  },
+  title: {
+    type: String,
+    required: true,
+  },
+  content: [
+    {
+      _id: Schema.Types.ObjectId,
+      type: String,
+      contentFragment: {
+        type: Schema.Types.Mixed,
+        required: true,
+      },
+    },
+  ],
+  likes: {
+    type: [Schema.Types.ObjectId],
+    ref: 'User',
+  },
+  private: {
+    type: Boolean,
+  },
+});
+
+const Blogs = mongoose.model('Blogs', BlogsSchema);
+module.exports = Blogs;
```

Comment:
export default should be used. We are following the ES6 format.

Reply:
Sure.